components/registry-facade-api/imagespec.proto

```diff
@@ -12,12 +12,14 @@ message ImageSpec {
     string ide_ref = 2;
     // content_layer describe the last few layers which provide the workspace's content
     repeated ContentLayer content_layer = 3;
-    // desktop_ide_ref points to an image denotign the desktop IDE to use
+    // DEPRECATED desktop_ide_ref points to an image denotign the desktop IDE to use
     string desktop_ide_ref = 4;
```

Comment:
Deprecating fields in `proto` is best done using `reserved` if we control both sides of the API (consumer and provider). Otherwise using the [deprecated field option](https://developers.google.com/protocol-buffers/docs/proto3) would be preferable.

```suggestion
    // Was used for desktop_ide_ref points to an image denotign the desktop IDE to use
    reserved 4;
```